configure

```diff
@@ -2878,6 +2878,18 @@ toolchain "aarch64_linux_gnu"
     set_toolset "ar" "aarch64-linux-gnu-ar" "ar"
 toolchain_end
 
+# emcc toolchain (wasm32)
+toolchain "wasm32_unknown_wasm"
```

Comment:
please rename to `emcc` toolchain. Consistent with xmake's toolchain naming. https://github.com/xmake-io/xmake/tree/master/xmake/toolchains/emcc

Reply:
Done.